packages/accordion/src/accordion.js

```diff
@@ -156,7 +160,7 @@ export default class Accordion {
 
 			// Make sure the selected element is a header and a direct descendant of the current accordionArea
 			if (
-				selectedElement.classList.contains('accordion-header') &&
+				selectedElement.classList.contains(headerClass) &&
```

Comment:
Looks like you meant `this.settings.headerClass` headerClass here is not defined.